### ISSUE TITLE
[JENKINS-49657] Make ath-container able to run tests using chrome

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/acceptance-test-harness#2462]
<p>Currently the ATH provides a <a href="https://github.com/jenkinsci/acceptance-test-harness/blob/master/src/main/resources/ath-container/Dockerfile" class="external-link" target="_blank" rel="nofollow noopener">docker container</a> quite useful to run it in a repeatable way both locally and in CI. However, the container does not contain the necessary conf and packages to run tests using a chrome browser.</p>

<p>As chrome is one of the supported platforms for Jenkins it will be interesting to have this container able to launch tests using a chrome browser</p>

---
<details><summary><i>Originally reported by <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/rarabaolaza.png" title="rarabaolaza's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=rarabaolaza">rarabaolaza</a>, imported from: <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-49657" target="_blank">Make ath-container able to run tests using chrome</a></i></summary>
<i><ul>
<li><b>assignee</b>: <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/rarabaolaza.png" title="rarabaolaza's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=rarabaolaza">rarabaolaza</a>
<li><b>status</b>: Open
<li><b>priority</b>: Minor
<li><b>component(s)</b>: acceptance-test-harness
<li><b>label(s)</b>: newbie-friendly
<li><b>resolution</b>: Unresolved
<li><b>votes</b>: 0
<li><b>watchers</b>: 2
<li><b>imported</b>: 20251212-090250
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>Currently the ATH provides a <a href="https://github.com/jenkinsci/acceptance-test-harness/blob/master/src/main/resources/ath-container/Dockerfile" class="external-link" target="_blank" rel="nofollow noopener">docker container</a> quite useful to run it in a repeatable way both locally and in CI. However, the container does not contain the necessary conf and packages to run tests using a chrome browser.</p>

<p>As chrome is one of the supported platforms for Jenkins it will be interesting to have this container able to launch tests using a chrome browser</p></pre>
</details>
</details>

**Comment by Samufds** [comment]
Hi,
I’m new to contributing to Jenkins and interested in GSoC 2026. I saw this issue labeled good-first-issue and would like to work on it.
Is this still relevant and available? 
I’d be happy to help update the ath-container to support Chrome tests.
Thanks!

**Comment by MarkEWaite** [comment]
> Hi, I’m new to contributing to Jenkins and interested in GSoC 2026. I saw this issue labeled good-first-issue and would like to work on it. Is this still relevant and available? I’d be happy to help update the ath-container to support Chrome tests. Thanks!

I don't think that this is a good first issue.  There has been progress since the issue was first opened through pull requests like:

* https://github.com/jenkinsci/acceptance-test-harness/pull/2057
* https://github.com/jenkinsci/acceptance-test-harness/pull/2026
* https://github.com/jenkinsci/acceptance-test-harness/pull/2018
* https://github.com/jenkinsci/acceptance-test-harness/pull/1280
* https://github.com/jenkinsci/acceptance-test-harness/pull/2010

The larger step is identifying the tests that fail with Chrome and fixing those failures.
